### PR TITLE
Indicate if a user has 2fa enabled or not on user overview

### PR DESCRIPTION
**Pull request by jvanbaarsen**
In order to make security more obvious for administrators, we want to show if a
user has 2fa enabled or not.

**Screenshot**
image

### CODE REVIEW

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -1,5 +1,12 @@
 <%= content_tag :tr, id: dom_id(user), class: "user" do %>
   <td><%= user.email %></td>
+  <td>
+    <% unless user.two_factor_auth_enabled? %>
+      <span data-tooltip="Two-factor security not enabled">
+      <%= icon "warning" %>
```
Comment (joshuajansen) [inline app/views/users/_user.html.erb]: Missing 1 tab indentation

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -9,6 +9,7 @@
           <th>Email</th>
           <th></th>
           <th></th>
+          <th></th>
```
Comment (joshuajansen) [inline app/views/users/index.html.erb]: You could replace these 3 `th`s with 1 `<th colspan="3">`

**Comment by joshuajansen** [comment]
Tiny nits. But looks good :shipit:

File: app/views/users/index.html.erb (hunk 2 above)
Comment (jvanbaarsen) [inline app/views/users/index.html.erb]: Even if I want to make it explicit that there are a couple of columns? I always thought colspan was used if content was spanning over multiple columns, right?

File: app/views/users/_user.html.erb (hunk 1 above)
Comment (jvanbaarsen) [inline app/views/users/_user.html.erb]: Good catch :-)

File: app/views/users/index.html.erb (hunk 2 above)
Comment (joshuajansen) [inline app/views/users/index.html.erb]: Yeah, that should work :) `colspan` makes sure the `td` or `th` spans 3 columns.

Comment (jvanbaarsen) [inline app/views/users/index.html.erb]: Oh haha yeah I don't mean it won't work, but isn't that breaking symantics? (As in explicit HTML)

**Pull request merged**